infra/base-images/base-builder/Dockerfile

```diff
@@ -138,6 +140,10 @@ RUN cd $SRC && \
 COPY precompile_afl /usr/local/bin/
 RUN precompile_afl
 
+RUN git clone https://github.com/ossf/fuzz-introspector.git fuzz-introspector && \
```

Comment:
Why do we need to clone this again if it's already been cloned in base-clang?
I think putting this in /src is causing breakages.
Although currently we (oss-fuzz's maintainers) use /src, we probably shouldn't.

Reply:
addressed in  #7199